packages/ListBoxWithTags/src/renderTrigger.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Tag, { Tags } from "@paprika/tag";
+import * as sc from "./ListBoxWithTags.styles";
+
+const renderTrigger = ({
+  allOptionsAreSelected,
+  hasError,
+  idListBoxContent,
+  isDisabled,
+  isReadOnly,
+  onRemove,
+  placeholder,
+  renderTag,
+  selectedOptions,
+  size,
+  tagLabelKey,
+  t,
+}) => (...args) => {
+  const [, , , { dispatch, handleKeyDown, handleKeyUp, isOpen, propsForTrigger, refTrigger, types }] = args;
+
+  function handleClick(event) {
+    event.stopPropagation();
+
+    if (event.key === "Enter" || event.key === " ") return;
+    dispatch({ type: types.togglePopover });
+  }
+
+  const handleRemove = option => () => {
+    onRemove(option);
+  };
+
+  function renderCaret(isDisabled) {
+    return isOpen ? <sc.UpIcon isDisabled={isDisabled} /> : <sc.DownIcon isDisabled={isDisabled} />;
+  }
+
+  const triggerProps = propsForTrigger();
+  const a11yTextOptions = selectedOptions.map(item => {
+    return tagLabelKey === null ? item.label : item[tagLabelKey];
+  });
+  const a11yText = selectedOptions.length === 0 ? t("listBoxWithTags.placeholder") : a11yTextOptions.join(", ");
```

Comment:
Shouldn't we have a custom trigger here as well? 

```js
placeholder || t("listBoxWithTags.placeholder")
```

Reply:
Not sure what you mean.  The `<ListBoxWithTags>` doesn't actually support a custom trigger (at this point). It's the `ListBox` with a built in custom trigger and custom hooks.

Reply:
i think he meant use the custom placeholder in the a11yText if they provided one:

```
const a11yText = selectedOptions.length === 0 ? placeholder ||
t("listBoxWithTags.placeholder") : a11yTextOptions.join(", ");
```

Reply:
Because isn't it just a `<ListBox>` then?

Maybe @nahumzs has more insight here, but my impression is that this component basically _is_ the custom trigger with tags.

Reply:
> i think he meant use the custom placeholder in the a11yText if they provided one

OH. 🤦

Reply:
Was part of a ListBox before, but doesn't make sense to have two-component with a similar UI but different concepts in the same package, is difficult to apply proper semver for the two components without affecting the versioning of each other. That's the reason why is a different package. 

is that the question @mikrotron ?